index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <link rel="stylesheet" type="text/css" href="css\sanitize.css">
+        <link rel="stylesheet" type="text/css" href="css\styles.css">
+    </head>
+
+    <body>
+        <header>
+            <h1>My Notes Application</h1>
+        </header>
+        <main>
+            <div class="create_note">
+                <h2>Create a New Note</h2>
+                <form>
+                    <div>
+                        <label>
```

Comment:
```suggestion
                        <label for='create-note-title'>
```

MDN: https://developer.mozilla.org/es/docs/Web/HTML/Element/label